src/plugins/management/public/management_sections_service.ts

```diff
@@ -72,10 +93,10 @@ export class ManagementSectionsService {
       }
     });
 
-    return {
-      getSection: this.getSection,
-      getAllSections: this.getAllSections,
+    setSectionsServiceStartPrivate({
       getSectionsEnabled: () => this.getAllSections().filter((section) => section.enabled),
-    };
+    });
+
+    return {};
```

Comment:
Why not return the relevant contract on the start contract too?
I think it's more convenient this way, don't you think?

Reply:
Its not accurate to how the contract is used. This functionality is only used internally, not externally.